api/pkg/api/main.go

```diff
@@ -179,3 +180,22 @@ func getServiceNameFromUrl(inputUrl string) (string, string) {
 func CheckIsServiceIP(address string) bool {
 	return k8sResolver.CheckIsServiceIP(address)
 }
+
+// gives a rough estimate of the size this will take up in the db, good enough for maintaining db size limit accurately
+func getEstimatedEntrySizeBytes(mizuEntry models.MizuEntry) int {
```

Comment:
This is going to get out of sync with the real size as soon as we add more fields in the DB.
For example, `getEstimatedEntrySizeBytes` does not take `EstimatedSizeBytes` into account.